docs/_articles/en/v3.x/code-analyzer-vs-code-extension.md

```diff
@@ -50,13 +50,13 @@ Complete one of these options to perform a Code Analyzer (beta) scan.
 * To scan selected files or folders:
 
 1. Select a group of files or folders.
-2. Right click in the VS Code Explorer and select **SFDX: Scan selected files or folders with Code Analyzer**.
+2. Right click in the VS Code Explorer and select **SF: Scan selected files or folders with Code Analyzer**.
```

Comment:
The extension still uses "SFDX" in its command names instead of "SF". I'm not sure if that's something we should change. I'll ask the CLI team and get back to you, but for now, please revert this change.

Reply:
@teresa-allen-sfdc , I've talked it through with the relevant team, and the consensus is that the extension's messages should change. So _don't_ revert this change.

Reply:
Excellent @jfeingold35 . And you'll update the messages to SF for this release? Thank you!